packages/web-components/src/components/leaving-ibm/leaving-ibm-modal-body.ts

```diff
@@ -33,7 +33,7 @@ class DDSLeavingIbmModalBody extends BXModalBody {
     const { href } = this;
     return html`
       <p><slot></slot></p>
-      <slot name="supplemental"></slot>
+      <span><slot name="supplemental"></slot></span>
```

Comment:
@andysherman2121 Instead of adding a new `span` element you can use `::slotted` selector in the css.